counterpartylib/protocol_changes.json

```diff
@@ -104,4 +104,11 @@
         "block_index": 423888,
         "testnet_block_index": 0
     }
+    "burn_assets": {
```

Comment:
missing a comma before this line.  And we'll obviously need to pick a block for the next release.